tests/unit/test_job_file.py

```diff
@@ -279,6 +279,8 @@ def test_write_prelude(monkeypatch, fixture_get_platform, set_CYLC_ENV_NAME):
         expected += '\nexport CYLC_ENV_NAME=\'myenv\''
     expected += '\nexport CYLC_WORKFLOW_ID="test_write_prelude"'
     expected += '\nexport CYLC_WORKFLOW_INITIAL_CYCLE_POINT=\'20200101T0000Z\''
+    #expected += '\nexport CYLC_WORKFLOW_NAME="test_write_prelude"'
+    #expected += '\nexport CYLC_WORKFLOW_NAME_BASE="test_write_prelude"'
```

Comment:
confusing as comments - recommend removal

```suggestion
```

Reply:
Oops, I did not intend to leave those there - a left-over from my first attempt at adding the new variables to the test.